native_libs/src/parse_csv.c

```diff
@@ -131,30 +131,37 @@ void mat_delete(void **mat)
     free(mat);
 }
 
+void write_csv(const char *filename, char **mat, size_t rows, size_t cols){
+    FILE *fp = fopen(filename, "w+");
+    if(!fp){
+        fprintf(stderr, "%s can't open in %s\n", filename, __func__);
+        perror("fopen");
+        abort();
```

Comment:
Do not abort() in production code, it's not really nice. You'd want to propagate the error to Luna, preferably by returning some error code here and 0 if successful.